.github/workflows/build-plugin-zip.yml

```diff
@@ -328,7 +328,7 @@ jobs:
             - name: Setup Node
               uses: actions/setup-node@5b52f097d36d4b0b2f94ed6de710023fbb8b2236 # v3.1.0
               with:
-                  node-version-file: '.nvmrc'
+                  node-version-file: 'main/.nvmrc'
```

Comment:
I caught it when testing the new workflow on the forked Gutenberg repository:

<img width="1080" alt="Screenshot 2022-05-12 at 12 11 13" src="https://user-images.githubusercontent.com/699132/168047676-efe5b7eb-8a82-446d-89d5-f3f3cd4e99c8.png">

It's always risky to apply changes that can be tested on production once every 2 weeks 😅